tds/modules/document/model.py

```diff
@@ -16,7 +16,11 @@ class Asset(BaseModel):
 
     file_name: str = Field(description="The file name of the asset")
     metadata: Optional[dict[str, Any]] = Field(
-        description="(Optional) Unformatted metadata field that should contain the asset type of the object under a 'type' key. Supported types are currently: 'equations', 'images', and 'figures'"
+        description=(
+            "(Optional) Unformatted metadata field that should contain the "
+            "asset type of the object under a 'type' key. Supported types are currently:"
```

Comment:
@Sorrento110 I feel like having `type` at the top level along with `metadata` and `file_name` would be more explicit. Any reason to keep it in `metadata` only referenced in `metadata`'s description?

Reply:
@brandomr I don't think there's a reason for keeping it in the metadata other than the comment on the issue that had them in metadata. I can move them out to a top-level key on the asset.